key-manager/src/main/scala/org/bitcoins/keymanager/CreateKeyManagerApi.scala

```diff
@@ -0,0 +1,45 @@
+package org.bitcoins.keymanager
+
+import org.bitcoins.core.crypto.MnemonicCode
+import scodec.bits.BitVector
+
+/**
+  * @define initialize
+  *                    Initializes the wallet, generating a wallet seed.
+  *                    This seed should be displayed to the user, so they
+  *                    can write it down. They should also be prompted
+  *                    to confirm at least parts of the code.
+  * @define initializeWithEnt
+  *                           Initializes the with a user-provided seed,
+  *                           generating a wallet seed.
+  *                           This seed should be displayed to the user, so they
+  *                           can write it down. They should also be prompted
+  *                           to confirm at least parts of the code.
+  */
+trait CreateKeyManagerApi {
```

Comment:
nit: I personally prefer `KeyManagerCreationApi` but not too strong a feeling, I just find it really weird to find a verb at the start of a class name